.vsts-ci.yml

```diff
@@ -54,6 +54,18 @@ jobs:
       name: Build
       displayName: Build
       condition: succeeded()
+    - script: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 -warnaserror:0 -ci
+        eng/sendToHelix.proj
+        /t:Test
+        /p:TestOS=Windows_NT
+        /p:Configuration=$(_BuildConfig)
+        /p:HelixBuild=$(Build.BuildNumber)
+        /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/SendToHelix.binlog
+        /p:HelixAccessToken=$(HelixApiAccessToken)
```

Comment:
Will `HelixApiAccessToken` always be defined? What happens in PR builds were the variable group is not available?

Reply:
It is evaluated as empty. Which basically means that it won't be set.

Reply:
Ok, cause I thought when a variable wasn't defined, azure devops would replace it with the literal `$(...)` value.